app/src/main/java/com/nextcloud/talk/activities/CallActivity.kt

```diff
@@ -699,6 +729,10 @@ class CallActivity : CallBaseActivity() {
         binding!!.lowerHandButton.setOnClickListener { l: View? -> raiseHandViewModel!!.lowerHand() }
     }
 
+    private fun showPopupMenu() {
+        binding!!.popupMenu.visibility = View.VISIBLE
```

Comment:
popupMenu should be renamed to indicate what it is about. Lets call it `leaveCallPopupMenu` ?
Same for the function name.

Reply:
renamed it